services/common/type/chipSPA.js

```diff
@@ -5,12 +5,7 @@ export const chipSPA = {
   idLabel: "SPA ID",
   idRegex:
     "(^[A-Z]{2}-[0-9]{2}-[0-9]{4}-[a-zA-Z0-9]{1,4}$)|(^[A-Z]{2}-[0-9]{2}-[0-9]{4}$)",
-  idExistValidations: [
-    {
-      idMustExist: false,
-      errorLevel: "error",
-    },
-  ],
+  idMustExist: false,
```

Comment:
I feel like you did such a good job in reducing idExistValidations into a single boolean that I also wonder if now idMustExist and allowMultiplesWithSameId mean the same thing?? Just a thought for another time...

Reply:
So... I agree.... and I think they CAN be combined at this time.... BUT - I feel like we will eventually need to handle that Medicaid SPAs can only have one RAI/Response to RAI while CHIPs can have multiple.... though, that should be handled by CMS, but.... I guess I chickened out on merging that up.... :)